editor/modes/visual-editor/block.js

```diff
@@ -51,13 +53,28 @@ import {
 	getMultiSelectedBlockUids,
 } from '../../selectors';
 
-const { BACKSPACE, ESCAPE, DELETE, ENTER } = keycodes;
+const { BACKSPACE, ESCAPE, DELETE, ENTER, TAB, F10, UP, LEFT, DOWN, RIGHT } = keycodes;
 
 function FirstChild( { children } ) {
 	const childrenArray = Children.toArray( children );
 	return childrenArray[ 0 ] || null;
 }
 
+function queryFirstTabbableChild( elem ) {
+	let i = 0;
+	for ( i = 0; i < elem.childNodes.length; i++ ) {
+		const tabbableNode = queryFirstTabbable( { context: elem.childNodes[ i ] } );
```

Comment:
Do we need this loop at all? Or function for that matter? What's the difference between this and:

```js
const queryFirstTabbableChild = ( elem ) => queryFirstTabbable( { context: elem } );
```